admin_guide/http_proxies.adoc

```diff
@@ -0,0 +1,120 @@
+= Working with HTTP Proxies
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+:description: This should be a clear and concise description of the topic.
+:keywords: comma, separated, keywords, list
+
+toc::[]
+
+== Overview
+Production environments can deny direct access to the web and instead, have an HTTP or
+HTTPS proxy available. Configuring OpenShift builds and deployments to
+use these proxies is as simple as setting standard environment variables in
+configuration or JSON files.
+
+== Importing Image Streams
```

Comment:
Having these env vars in the /etc/sysconfig files does not [currently] affect imagestreams, deploymentConfigs etc. So, rather than "importing image streams", the env var settings in /etc/sysconfig/* are more something like "Component configuration" but you probably can suggest a better name. It should also explain that these settings should be added to /etc/sysconfig/openshift-node (in addition to openshift-master), and maybe it's worth merging in the "Proxying Docker Pull" section below in here to keep all the settings related to /etc/sysconfig together.

The image streams / templates should be updated separately, so I'd suggest a separate section.